src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlString.cs

```diff
@@ -1,55 +1,31 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.IO;
-using Microsoft.AspNet.Html.Abstractions;
-using Microsoft.Framework.WebEncoders;
-
 namespace Microsoft.AspNet.Mvc.Rendering
 {
     /// <summary>
     /// String content which knows how to write itself.
     /// </summary>
-    public class HtmlString : IHtmlContent
+    public class HtmlString : StringHtmlContent
```

Comment:
Inheriting from `StringHtmlContent` makes that class more complex than necessary. `HtmlString` is the only user of `encodeOnWrite: false`. Implement `IHtmlContent` here and remove `encodeOnWrite` from `StringHtmlContent`.